### PR TITLE
Make the website look a little nicer in the WBE browser

### PR DESCRIPTION
**Pull request by pavpanchekha**
This PR uses a few weird parser tricks to make the browser look a little better in lab6.py, including overriding the (too dark) background for `<pre>` tags and also making code in pre tags not super small. I also made sure that, if the user implemented class selectors, that footnotes would appear in gold like they're supposed to.

### CODE REVIEW

File: www/book.css [hunk 1]
```diff
@@ -13,7 +28,11 @@ body {
 }
 
 pre, code { hyphens: none; -webkit-hyphens: none; }
-pre { font-size: 18px; overflow: auto; padding-left: 2ex; }
+pre {
+    font-size: 18px; overflow: auto; padding-left: 2ex;
+    background-color: #ddd; /* For WBE browser only */
```
Comment (chrishtr) [inline www/book.css:33]: Does the WBE browser properly ignore comments?

File: www/book.css [hunk 2]
```diff
@@ -30,7 +49,7 @@ header, h1, strong {
     color:black;
 }
 }
-h1 { font-size: 105%; margin: 1.5em 0 0; text-align: left; }
+h1 { font-size: 105%; font-weight: bold; margin: 1.5em 0 0; text-align: left; }
```
Comment (chrishtr) [inline www/book.css:52]: Why this change?

File: www/book.css [hunk 3]
```diff
@@ -180,35 +201,33 @@ body.main { padding: 0 4ex; }
     font-weight: bold; font-family: 'Vollkorn', 'Garamond', serif;
     text-align: center;
 }
-p
```
Comment (chrishtr) [inline www/book.css:183]: What was this p?

File: www/book.css (hunk 2 above)
Comment (pavpanchekha) [reply www/book.css:52]: Ah—our browser's default style sheet doesn't bold `h1` tags by default, figured this was safe to add since all other browsers do.

File: www/book.css (hunk 1 above)
Comment (pavpanchekha) [reply www/book.css:33]: No, or, uhh, sortof. It doesn't have handling for comments, but when it sees a comment it treats it as a syntax error which causes it to eat everything up to the next semicolon. So this comment sort of deletes the next property (`background: transparent`). This doesn't change how our browser behaves because it doesn't support `background` anyway.
